pbm/restore/logical.go

```diff
@@ -1330,7 +1331,11 @@ func (r *Restore) applyOplog(ctx context.Context, ranges []oplogRange, options *
 }
 
 func (r *Restore) snapshot(input io.Reader, cloneNS snapshot.CloneNS, excludeRouterCollections bool) error {
-	rf, err := snapshot.NewRestore(r.brief.URI, r.cfg, cloneNS, r.numParallelColls, excludeRouterCollections)
+	rf, err := snapshot.NewRestore(r.brief.URI,
```

Comment:
```suggestion
	rf, err := snapshot.NewRestore(
	r.brief.URI,
```
optionally but good for style consistency: keep each argument on a separate line so it is easy to see all of them. except for that `ctx`